datafusion/core/src/datasource/physical_plan/csv.rs

```diff
@@ -656,7 +682,14 @@ mod tests {
         let mut config = partitioned_csv_config(file_schema, file_groups)?;
         config.projection = Some(vec![0, 2, 4]);
 
-        let csv = CsvExec::new(config, true, b',', file_compression_type.to_owned());
+        let csv = CsvExec::new(
+            config,
+            true,
+            b',',
+            b'"',
+            None,
```

Comment:
I wonder if we can reduce the number of arguments to `CsvExec::new()` somehow (like maybe pass in the config 🤔 )